src/api/profile.rs

```diff
@@ -18,9 +21,22 @@ pub async fn get_profile(
     mut conn: Connection<Db>,
     id: u32,
 ) -> Result<Json<ApiResponse<Profile>>, GenericError> {
+    let time = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
```

Comment:
why is there an unwrap here? add the error .duration_since returns to GenericError and replace .unwrap with `?`

Reply:
replacing .unwrap with a ? does not work. Also this thing is unable to fail, since it just gets your systemtime, if its unable to do that, you have bigger things to worry about